scripts/on-release.js

```diff
@@ -8,8 +8,8 @@ const shell = require('shelljs');
 const version = require('../package.json').version;
 
 const starlarkModule = process.argv[2];
-if (starlarkModule !== 'build_bazel_rules_nodejs' && starlarkModule != 'rules_nodejs') {
-  throw new Error("first argument must be module to release (build_bazel_rules_nodejs or rules_nodejs)")
+if (starlarkModule != 'rules_nodejs') {
```

Comment:
the script can be removed and replaced with our standard release.yaml

https://github.com/aspect-build/rules_jasmine/blob/main/.github/workflows/release.yml
+
https://github.com/aspect-build/rules_jasmine/blob/main/.github/workflows/release_prep.sh